openapi.yaml

```diff
@@ -1521,6 +1585,25 @@ components:
           description: The fully-qualified property name, including the 'namespace'.
         value:
           type: string
+    GalasaProperty:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          properties:
+            namespace:
+              type: string
+              description: The namespace the property resides in
+            name:
+              type: string
+              description: The name of the property
+        data:
+          properties:
+            value:
+              type: string
+              description: The value of the property
```

Comment:
Can it be an empty string ? What is the max length of this value ?
Later we will be saying that it's base64-encoded also.